HeRoBackEnd/Controllers/AuthController.cs

```diff
@@ -72,7 +72,7 @@ public async Task<IActionResult> LogOut()
         /// <response code="400">Invalid email or password or user already exist</response>
         [HttpPost]
         [Route("Auth/CreateNewUser")]
-        [Authorize(Policy = "AdminRequirment")]
+        //[Authorize(Policy = "AdminRequirment")]
```

Comment:
Why commented?

Reply:
Right, I just wanted to create a user without admin privilages. Have to fix that.